upstream-wpt-webhook/init.sls

```diff
@@ -0,0 +1,53 @@
+{% from tpldir ~ '/map.jinja' import webhook %}
+
+include:
+  - python
+
+upstream-wpt-webhook:
+  virtualenv.managed:
+    - name: /home/servo/upstream-wpt-sync-webhook/_venv
+    - venv_bin: virtualenv-3.5
+    - python: python3
+    - system_site_packages: False
+    - require:
+      - pkg: python3
+      - pip: virtualenv
+  pip.installed:
+    - pkgs:
+      - git+https://github.com/servo-automation/upstream-wpt-sync-webhook@{{ webhook.rev }}
```

Comment:
It would be nice if you publish to PyPI - the Salt state does much better with PyPI packages than git packages.

Reply:
Never mind, no longer needed as #593 is fixed.